src/drupal8/helm/app/templates/application/php-fpm/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: php-fpm
+  labels:
+    app.service: php-fpm
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.service: php-fpm
+  template:
+    metadata:
+      labels:
+        app.service: php-fpm
+    spec:
+      containers:
+      - env:
+        {{ range $key, $value := .Values.environment }}
+        - name: {{ $key }}
+          value: "{{ $value }}"
+        {{ end }}
+        image: {{ .Values.docker.image.fpm }}
+        imagePullPolicy: Always
+        name: php-fpm
+        ports:
+        - containerPort: 9000
+        resources:
+          limits:
+            memory: {{ .Values.resources.limits.memory.php_fpm }}
+          requests:
+            memory: {{ .Values.resources.requests.memory.php_fpm }}
+        readinessProbe:
+          tcpSocket:
+            port: 9000
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        volumeMounts:
+        - name: php-fpm-persistent-storage
+          mountPath: /app/docroot/sites/default/files
```

Comment:
I assume needs to be mounted on the nginx container as well for web access?

Reply:
I've asked around, and generally nginx does need access. My PR can be used with for example the nfs-server-provisioner chart to provide a test ReadWriteMany volume